uat/custom-components/client-python-paho/scripts/black_run.bat

```diff
@@ -0,0 +1,7 @@
+@REM -------------------------------
+@REM Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+@REM SPDX-License-Identifier: Apache-2.0
+@REM -------------------------------
+
+@echo off
+black --line-length 79 --verbose --exclude="dev-env|grpc_client_server/grpc_generated" %~dp0\..\src

```

Comment:
please use 120 line length.

Reply:
Updated. Also added 120 line lengths for black and pylint.